src/lib/components/top-nav.svelte

```diff
@@ -30,17 +30,19 @@
   }
 </script>
 
-<div
+<nav
```

Comment:
Looks like clicking on data encoder settings no longer shows the settings unless you are at the top of the page 

![Screenshot 2023-08-10 at 10 56 50 AM](https://github.com/temporalio/ui/assets/15069288/0a7438fa-ff9b-4d31-b151-d186cbb20de2)